content/en/cosign/openid_signing.md

```diff
@@ -35,6 +35,25 @@ To log in and set up your OIDC Identity, follow these steps:
 
 Currently, Sigstore supports Microsoft, Google, and GitHub. As an alternative, you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself by setting its value to that of the identity token. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
 
+### The signing, witnessing, and verifying process
+
+Identity-based signing is the default because managing and distributing keys is hard to do securely, and developers are very likely to already have a trusted identity rather than a signing key. When using Sigstore's defaults for signing and verification, the process of signing is as follows:
```

Comment:
I think this can be a little tentative, and I would suggest "challenging" or "difficult" over "hard"

```suggestion
Identity-based signing is the default because managing and distributing keys can be challenging to do securely, and developers are very likely to already have a trusted identity rather than a signing key. When using Sigstore's defaults for signing and verification, the process of signing is as follows:
```